### PR TITLE
Fix/4684 frogot password initial email value prop

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes BLUI-4684 .

#### Changes proposed in this Pull Request:

- Fix invalid `initialEmailValue` prop should disable `Submit` button
-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2023-09-15 at 11 03 39 AM

#### To Test:

- Update AppRouter file with below code
`<ForgotPasswordScreen initialEmailValue='test'/>` and go to http://localhost:3000/forgot-password 

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Prettier is failing.
Please run `yarn prettier`

File: login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx [hunk 1]
```diff
@@ -55,9 +55,11 @@ export const ForgotPasswordScreenBase: React.FC<React.PropsWithChildren<ForgotPa
     const instructionsProps = props.WorkflowCardInstructionProps || {};
     const actionsProps = props.WorkflowCardActionsProps || {};
 
-    const [isEmailValid, setIsEmailValid] = useState(emailValidator(initialEmailValue) ?? false);
-    const [emailError, setEmailError] = useState('');
-    const [shouldValidateEmail, setShouldValidateEmail] = useState(false);
+    const validateEmail = (): boolean => typeof emailValidator(initialEmailValue) === 'string' ? false : true;
```
Comment (surajeaton) [inline login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx:58]: We can minimize this to:
    const validateEmail = (): boolean => typeof emailValidator(initialEmailValue) !== 'string';

Comment (ektaghag-eaton) [reply login-workflow/src/screens/ForgotPasswordScreen/ForgotPasswordScreenBase.tsx]: Done. Please review.

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**